Loops/Problem14/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Problem14
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int quotient = int.Parse(Console.ReadLine());
+
+            Stack<string> result = new Stack<string>();
+
+            while (quotient != 0)
+            {
+                int remainder = quotient % 16;
+                quotient /= 16;
+
+                result.Push(CheckRemainder(remainder));
```

Comment:
Can we get the right characters without using CheckReminder() at all? Try Convert.ToChar()